test/acceptance/features/bindAppToService.feature

```diff
@@ -307,3 +307,30 @@ Feature: Bind an application to a service
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-a-d-c" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-a-d-c" should be changed to "True"
         And Secret "binding-request-a-d-c" contains "SOME_KEY" key with value "SOME_VALUE:5432:db-demo-a-d-c"
+
+    Scenario: Bind an imported Node.js application to Etcd database
+        Given Etcd operator running
+        * Etcd cluster "etcd-cluster-example" is running
+        * Nodejs application "node-todo-git" imported from "quay.io/pmacik/node-todo" image is running
+        When Service Binding is applied to connect the database and the application
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+              name: binding-request-etcd
+            spec:
+              application:
+                group: apps
+                version: v1
+                resource: deployments
+                name: node-todo-git
+              services:
+                - group: etcd.database.coreos.com
+                  version: v1beta2
+                  kind: EtcdCluster
+                  name: etcd-cluster-example
+              detectBindingResources: true
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-etcd" should be changed to "True"
+        And deployment must contain intermediate secret "binding-request-etcd"
```

Comment:
Application endpoint step got introduced in this PR: `https://github.com/redhat-developer/service-binding-operator/pull/613/files#diff-ec891afbdb0cf5fd075cbf5ca535a45fR118 but it is not used anywhere. IMHO, instread of checking that deployment must contain the secret, checking if application endpoint works is actually ultimate check and we should apply that instead.

Reply:
Good point, I've updated and squashed...